onnxruntime/python/tools/quantization/README.md

```diff
@@ -95,7 +101,7 @@ onnx.save(quantized_model, 'path/to/the/quantized_model.onnx')
                                })
     ```
 
-## Options
+### Options
```

Comment:
couldn't leave a comment on existing code, but line 82, 83 refer to input_quantization_params and output_quantization_params options which don't exist.